language/java/develop.md

```diff
@@ -108,7 +108,7 @@ services:
     command: ./mvnw spring-boot:run -Dspring-boot.run.profiles=mysql -Dspring-boot.run.jvmArguments="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8000"
```

Comment:
As mentioned during the blog post review this part should be rewrite, we usually use multi-stage build (see the one used in the run-tests.md file) and use a specific target to avoid this kind of complex compose file
Maybe moving this [section](https://deploy-preview-15269--docsdocker.netlify.app/language/java/run-tests/#multi-stage-dockerfile-for-development) here, and just add the new test target in the test section could be a good improvement

Reply:
Thanks @glours. The multi-stage Dockerfile section is now moved into the development topic in https://github.com/docker/docker.github.io/pull/15269/commits/036c8924d37c2af4ab32b6955b5bd033f3d5e889.